src/Microsoft.Fast.Components.FluentUI/Components/NumberField/FluentNumberField.razor.cs

```diff
@@ -59,13 +59,13 @@ public partial class FluentNumberField<TValue> : FluentInputBase<TValue>
     /// Gets or sets the maximum value
     /// </summary>
     [Parameter]
-    public string? Max { get; set; } = GetMaxValue();
+    public string? Max { get; set; } = InputHelpers<TValue>.GetMaxValue();
```

Comment:
Why to force to have a `Min` and `Max` value? Why not to keep the ` null` default value?

Reply:
Good question? Just can't think of the answer anymore 😁
Will leave it like this for v2 (to not break anything) and revise for v3

Reply:
OK, but why to force to have a default value?
Without set a specific `Min=null` the HTML generated code will contain `min="..."`.
But you can use the `fluent-number-field` component without these min/max attributes.

Maybe check the generated "string" only if the dev set a value.
In addition, you can set `null` :-)

Reply:
Will change for v3!

Reply:
Perfect :-)